DeskAPIClient/DeskAPIClientTests/DSAPICompanyTests.m

```diff
@@ -45,6 +45,18 @@ - (void)setUp
     _client = [DSAPITestUtils APIClientBasicAuth];
 }
 
+- (void)testEnhancementsEnabled
+{
+    [DSAPICompany enhancementsEnabledWithParameters:nil client:self.client queue:self.APICallbackQueue success:^(BOOL enabled) {
+        XCTAssertTrue(enabled);
```

Comment:
do we need assertion here? I think this method should only fail when error occurred, otherwise if enabled true or false it should succeed.